media_driver/linux/common/os/i915/include/intel_chipset.h

```diff
@@ -220,15 +220,21 @@
 
 #define PCI_CHIP_COFFEELAKE_S_GT1_1     0x3E90
 #define PCI_CHIP_COFFEELAKE_S_GT1_2     0x3E93
+#define PCI_CHIP_COFFEELAKE_S_GT1_3     0x3E99
```

Comment:
For this device Id reference will be:
`c99d783 ("drm/i915/cfl: Adding more Coffee Lake PCI IDs.")`

Please, check that you add other device ids from the above kernel commit, i.e.: 0x3e99, 0x3E9A (you miss it), 0x3EA0, 0x3EA1, 0x3EA2, 0x3EA3, 0x3EA4, 0x3EA9

Reply:
Added these device IDs.